CHECK_POWER_COMMANDS/Backstop_File_Class.py

```diff
@@ -257,3 +257,6 @@ def insert_errors( self, lr_file, violations_list):
         outfile = open(lr_file+'.ERRORS', 'w')
         outfile.writelines(ALR_lines)
         outfile.close()
+
+        # OK now copy the ACIS-LoadReview.txt.ERRORS file into ACIS-LoadReview.txt
+        os.system('mv ACIS-LoadReview.txt.ERRORS ACIS-LoadReview.txt')
```

Comment:
Use `shutil.move` (https://docs.python.org/3/library/shutil.html#shutil.move).  Basically you should never use `os.system` because the exception handling is poor.  If you really need to run an external command use `subprocess` or `Ska.Run`.

Reply:
I agree with Tom here, we should avoid system calls